README.md

```diff
@@ -69,7 +69,7 @@ You will see the link on the top of each task.
 
 ### Ready?
 
-➡️ Go forward to [Task 0](test00.md)
+➡️ Go forward to [Task 0](tasks/git-intro.md)
```

Comment:
`Task 0` name wouldn't remove problem, described in the issue. Probably we should get rid of numbering here as well.

Reply:
I thought about it too but I was not sure is it necessary. We still need enumeration of tasks to make it clear for students.